File: SISPU/client-app/src/app/classes/grouptimetable.ts

import { Teacher} from './teacher';
import { Auditory} from './auditory';
import { Lesson} from './lesson';
import { Day} from './day';

export class Grouptimetable {

    group_name: string = "";
    group_department: string ="";
    group_employment: string ="";
    group_tasks: number = 0;
    group_advice: number = 0;

    constructor() 
    {   this.group_name = "";
        this.group_department ="";
        this.group_employment ="";
        this.group_tasks = 0;
        this.group_advice = 0;
    }
}

File: SISPU/Models/App/GroupTimetable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SISPU.Models
{

    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using Microsoft.AspNetCore.Identity;
    using Microsoft.AspNetCore.Identity.EntityFrameworkCore;

    public class GroupTimetable
    {
        public string GroupName { get; set; }

        public string GroupDepartment { get; set; }

        public string GroupEmployment { get; set; }

        public int GroupTasks { get; set; }

        public string[] GroupTasks2 { get; set; }

        public int GroupAdvice { get; set; }

        public string[] GroupAdvice2 { get; set; }

        public string[] Answers { get; set; }

        public string[] Recs { get; set; }


    }
}
